packages/web-components/fast-foundation/src/combobox/combobox.spec.ts

```diff
@@ -561,4 +561,38 @@ describe("Combobox", () => {
 
         expect((element as Combobox).control.value).to.eq("three");
     });
+
+    it("should perform case-sensitive search when case-sensitive attribute it TRUE", async () => {
+        const { connect, element, option3 } = await setup();
+
+        await connect();
+
+        element.caseSensitive = true;
+        element.value = "THREE";
+
+        await DOM.nextUpdate();
+
+        expect(element.control).to.exist;
+
+        // make sure lowercase value was NOT selected
+        expect((element as Combobox).control.value).to.eq("THREE");
+        expect(option3.selected).to.be.false;
+    });
+
+    it("should perform case-insensitive search when case-sensitive attribute is FALSE", async () => {
+        const { connect, element, option3 } = await setup();
+
+        await connect();
+
+        element.caseSensitive = false;
```

Comment:
Maybe structure this one to be default behavior instead of explicitly setting the attribute false.

Reply:
It actually is the default behaviour. I'm just defensively setting it to false. Honestly, line 587 is unnecessary.

Reply:
I see it's the default. I mean to test the actual default case, as in not setting it defensively. Then we'd know if something broke or changed in the default somehow.